Tests/SonarScanner.MSBuild.PreProcessor.Tests/TargetsInstallerTests.cs

```diff
@@ -164,6 +164,43 @@ public void InstallLoaderTargets_Running_Under_LocalSystem_Account()
             act.Should().NotThrow();
         }
 
+        [TestMethod]
+        public void InstallLoaderTargets_ExceptionsOnCopyAreSuppressed()
+        {
+            // Arrange
+            var targetsInstaller = new TargetsInstaller(this.logger, this.msBuildPathSettingsMock.Object,
+                this.fileWrapperMock.Object, this.directoryWrapperMock.Object);
+
+            bool exceptionThrown = false;
+
+            var sourcePathRegex = "bin\\\\(?:debug|release)\\\\targets\\\\SonarQube.Integration.targets";
+            this.fileWrapperMock
+                .Setup(x => x.ReadAllText(It.IsRegex(sourcePathRegex, RegexOptions.IgnoreCase)))
+                .Returns("sourceContent");
+
+            this.fileWrapperMock
+                .Setup(x => x.Exists("c:\\project\\bin\\targets\\SonarQube.Integration.targets"))
+                .Returns(false);
+
+            this.fileWrapperMock
+                .Setup(x => x.Copy(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<bool>()))
+                .Callback(() =>
```

Comment:
I think there is a `Throws` method, then you could use `Verify` to test if it was called. The current solution is good enough, though.